src/growth-pandora.R

```diff
@@ -217,6 +222,13 @@ parameterFilePlaceHolders <- list(
 
 ## Extract relevant parameters ----
 
+# Check if multithreading enabled
+if (Multithreading$EnableMultithreading) { # default is TRUE
+  numThreads <- Multithreading$ThreadsPerIteration
```

Comment:
remove if/else statement (lines 225-230) and just keep this line: 
`numThreads <- Multithreading$ThreadsPerIteration`